d2bs/kolbot/libs/common/Pather.js

```diff
@@ -1578,6 +1578,81 @@ MainLoop:
 		return true;
 	},
 
+	/*
+	    Pather.TeamCheck();
+		Make Leader communication to others. Add by WhyY0416
+	*/
+	TeamCheck: function (phase) {
+	    var MemberCount = 0, 
+		Member = [], 
+		timeOut = 600;    //How to put this in character's config ?
```

Comment:
`Config.js` holds the default Config variables that are overridden in the character config files. You can put this in both those places, and the one in `Config.js` will be used as a default if the character config doesn't have it

Reply:
somthing like `Config.TeamCheck.Timeout` or whatever